app/javascript/lib/honeycrisp.js

```diff
@@ -0,0 +1,323 @@
+var incrementer = (function() {
+    var i = {
+        increment: function(input) {
+            var max = parseInt($(input).attr('max'));
+            var value = parseInt($(input).val());
+            if(max != undefined) {
+                if(value < max) {
+                    $(input).val(value+1);
+                }
+            }
+            else {
+                $(input).val(parseInt($(input).val())+1);
+            }
+        },
+        decrement: function(input) {
+            var min = parseInt($(input).attr('min'));
+            var value = parseInt($(input).val());
+            if(min != undefined) {
+                if(value > min) {
+                    $(input).val(value-1);
+                }
+            }
+            else {
+                $(input).val(value-1);
+            }
+
+        },
+        init: function() {
+            $('.incrementer').each(function(index, incrementer) {
+                var addButton = $(incrementer).find('.incrementer__add');
+                var subtractButton = $(incrementer).find('.incrementer__subtract');
+                var input = $(incrementer).find('.text-input');
+
+                $(addButton).click(function(e) {
+                    i.increment(input);
+                });
+
+                $(subtractButton).click(function(e) {
+                    i.decrement(input);
+                });
+            });
+        }
+    }
+    return {
+        init: i.init
+    }
+})();
+
+var radioSelector = (function() {
```

Comment:
This whole function is very funny.

It looks like we are mutating DOM class lists to avoid having to write CSS?? Maybe we can delete it someday by using SASS mixins more aggressively.

I know that's a honeycrisp issue, not a vita-min issue, so... whatever. :)